cli/src/index.ts

```diff
@@ -25,4 +26,11 @@ program
         runGenerate(options.pattern, options.output, !!options.verbose);
     });
 
+program
+    .command('validate')
+    .requiredOption('-p, --pattern <pattern>', 'The location of the JSON Schema Pattern')
```

Comment:
I think we should use a consistent description of this field and the instantiation field - since it is going to be used in all 3 of the existing commands (I've yet to add it to visualization, but I will)